README.md

```diff
@@ -26,15 +22,14 @@ angular.module('App', [
 
 ## Related
 
-* [Web Control Panel UI](https://github.com/ovh-ux/ovh-manager-web).
-* [Exchange Module](https://github.com/ovh-ux/ovh-module-exchange).
-* [Office Module](https://github.com/ovh-ux/ovh-module-office).
-* [Sharepoint Module](https://github.com/ovh-ux/ovh-module-sharepoint).
+* [ovh-manager-web](https://github.com/ovh-ux/ovh-manager-web) - OVH Control Panel Web UI
+* [ovh-module-exchange](https://github.com/ovh-ux/ovh-module-exchange) - Web Module Exchange
+* [ovh-module-office](https://github.com/ovh-ux/ovh-module-office) - Web Module Office
+* [ovh-module-sharepoint](https://github.com/ovh-ux/ovh-module-sharepoint) - Web Module Sharepoint
 
 ## Contributing
 
-Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/ovh-manager-telecom/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/ovh-manager-telecom/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
-
+Always feel free to help out! Whether it's [filing bugs and feature requests](https://github.com/ovh-ux/ovh-module-emailpro/issues/new) or working on some of the [open issues](https://github.com/ovh-ux/ovh-module-emailpro/issues), our [contributing guide](CONTRIBUTING.md) will help get you started.
```

Comment:
... [filling ... (typo)

Reply:
I'm not sure about it (https://www.thefreedictionary.com/filing)

Reply:
Ok I didn't know